--- o.GNS3-0.3.2/setup.py	2007-10-29 12:12:35.000000000 +0100
+++ n.GNS3-0.3.2/setup.py	2007-11-04 14:50:11.000000000 +0100
@@ -10,7 +10,7 @@
 
 setup( # Distribution meta-data
         name = "gns3",
-        version = "0.2alpha",
+        version = "0.3.2",
         description = "A graphical frontend to dynamips",
         author = "Jeremy Grossmann, David Ruiz, Romain Lamaison, Aurelien Levesque, Xavier Alt",
         author_email = "contact@gns3.net",
